definition/data/sheets/CaseEventToComplexTypes.json

```diff
@@ -31,5 +31,15 @@
     {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "otherDirectionHeaders","CaseFieldID": "otherDirections","EventElementLabel": "Direction header","EventHintText": "e.g. Upload contracts /Confirm documents are original","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
     {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "directionComment","CaseFieldID": "otherDirections","EventElementLabel": "Direction comment","EventHintText": "","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
     {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "sendBy","CaseFieldID": "otherDirections","EventElementLabel": "Response deadline","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "OPTIONAL"},
-    {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "forParty","CaseFieldID": "otherDirections","EventElementLabel": "Direction is for","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "MANDATORY"}
+    {"ID": "OtherDirection","CaseEventID": "DrawJudgesOrder","ListElementCode": "forParty","CaseFieldID": "otherDirections","EventElementLabel": "Direction is for","EventHintText": "","FieldDisplayOrder": "","DisplayContext": "MANDATORY"},
+    {"ID": "StaffDocument","CaseEventID": "ReferToJudge","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentName","EventElementLabel": "Name","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "StaffDocument","CaseEventID": "ReferToJudge","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentType","EventElementLabel": "Type","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
+    {"ID": "StaffDocument","CaseEventID": "ReferToJudge","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentLink","EventElementLabel": "Link","FieldDisplayOrder": 3,"DisplayContext": "MANDATORY"},
+    {"ID": "StaffDocument","CaseEventID": "ReferToJudge","CaseFieldID": "staffUploadedDocuments","ListElementCode": "receivedDatetime","EventElementLabel": "Document received on","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "StaffDocument","CaseEventID": "ManageDocuments","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentName","EventElementLabel": "Name","FieldDisplayOrder": 1,"DisplayContext": "OPTIONAL"},
+    {"ID": "StaffDocument","CaseEventID": "ManageDocuments","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentType","EventElementLabel": "Type","FieldDisplayOrder": 2,"DisplayContext": "MANDATORY"},
+    {"ID": "StaffDocument","CaseEventID": "ManageDocuments","CaseFieldID": "staffUploadedDocuments","ListElementCode": "documentTypeOther","EventElementLabel": "What type of document is it?","FieldDisplayOrder": 2,"DisplayContext": "OPTIONAL"},
```

Comment:
Are the FieldDisplayOrders correct on these lines?

Reply:
Probably not. We will have to double check this once ROC-7242 is done.